test/release/examples/benchmarks/shootout/meteor-fast.chpl

```diff
@@ -269,13 +269,14 @@ proc searchParallel(in board=0, in pos=0, used=0, placed=0, firstPiece=0) {
       if allMasks[currentMask] {
         const mask = allMasks[currentMask];
 
-        const currentSolution: [piecesDom] int =
-          [firstPiece, mask, 0, 0, 0, 0, 0, 0, 0, 0];
 
-        begin with (in board, in pos)
+        begin with (in board, in pos) {
+          const currentSolution: [piecesDom] int =
+            [firstPiece, mask, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
The issue is that according to our specified array semantics, the local variable currentSolution can be destroyed (when the function exits) before the begin runs. There may be some algorithm reason that is not possible (that I have not noticed) though.